scripts/install

```diff
@@ -103,6 +105,18 @@ popd
 # Set permissions to app files
 chown -R $app:root "$install_dir"
 
+#=================================================
+## SET STANDARD SETTINGS FROM DEFAULT CONFIG
+#=================================================
+disable_federation="false"
+guests_disabled="true"
+registration_shared_secret=""
```

Comment:
```suggestion
registration_shared_secret=""
enable_registration_captcha="false"
```

Reply:
not needed, see https://github.com/Gredin67/dendrite_ynh/blob/patch-1/scripts/install#L22-L26

Reply:
When I tried the upgrade on my server, it failed due to this setting being `true` but keys being empty.

Reply:
right. To avoid spamming to and from the federation, I deactivated federation as long as no Captcha verification has been activated (just hiding the option in the config panel). I added some instructions. We could probably copy some instructions from the config panel to the README too.